src/Adyen/Service/Checkout.php

```diff
@@ -216,18 +222,33 @@ public function ordersCancel($params, $requestOptions = null)
      * @param array $params
      * @param array|null $requestOptions
      * @return mixed
-     * @throws \Adyen\AdyenException
+     * @throws AdyenException
      */
     public function donations($params, $requestOptions = null)
     {
         return $this->donations->request($params, $requestOptions);
     }
 
+    /**
+     * @param CreateCheckoutSessionRequest $createCheckoutSessionRequest
+     * @param array|null $requestOptions
+     * @return mixed
+     * @throws AdyenException
+     */
+    public function sessionsTypes(
```

Comment:
This feature is still a work in progress that should not be part of the public API yet. Move this code into unit tests.

Reply:
I thought I'd add it here so people can get an idea of how it would look like, and then remove it again in the next ticket for service automation. Although I see your point, maybe it's better leave it out of this PR for now (also the checkout models itself) and just add it in the next?

Reply:
done